app/controllers/CheckYourAnswersController.scala

```diff
@@ -72,7 +72,7 @@ class CheckYourAnswersController @Inject()(
       val tradingNameSummaryRow = TradingNameSummary.checkAnswersRow(request.userAnswers, waypoints, thisPage)
 
       val websiteSummaryRow = WebsiteSummary.checkAnswersRow(request.userAnswers, waypoints, thisPage)
-      val previouslyRegisteredSummaryRow = PreviouslyRegisteredSummary.row(request.userAnswers, waypoints, thisPage)
+      val previouslyRegisteredSummaryRow = PreviouslyRegisteredSummary.row(request.userAnswers, waypoints, thisPage, lockEditing = false)
```

Comment:
Don't want to lock editing on CheckYourAnswers, Only on ChangeYourAnswers when the the answer is true